### ISSUE TITLE
Support selfhosted options

### ISSUE THREAD

**Issue by vistar** [omnivore-app/obsidian-omnivore#269]
Add URL and API of the server.

**Comment by TAT-Hins** [comment]
Yeah same request here. Omnivore switched to completely self-hosted for a long time, and I think it would be great to support self-hosted for related plugins.
